boards/FSM/Src/Io/Io_FlowMeters.c

```diff
@@ -0,0 +1,81 @@
+#include <assert.h>
+#include "main.h"
+#include "Io_SharedFreqOnlyPwmInput.h"
+#include "Io_FlowMeters.h"
+
+struct FlowMeters_Context
+{
+    struct FreqOnlyPwmInput *pwm_input;
+    TIM_HandleTypeDef *      htim;
+    HAL_TIM_ActiveChannel    active_channel;
+};
+
+static struct FlowMeters_Context primary_flow_meter, secondary_flow_meter;
+
+void Io_FlowMeters_Init(TIM_HandleTypeDef *htim)
+{
+    assert(htim != NULL);
+
+    primary_flow_meter.htim   = htim;
+    secondary_flow_meter.htim = htim;
+
+    primary_flow_meter.active_channel   = HAL_TIM_ACTIVE_CHANNEL_1;
+    secondary_flow_meter.active_channel = HAL_TIM_ACTIVE_CHANNEL_2;
+
+    primary_flow_meter.pwm_input = Io_SharedFreqOnlyPwmInput_Create(
+        htim, TIMx_FREQUENCY / TIMx_PRESCALER, TIM_CHANNEL_1,
```

Comment:
What is `TIMx_FREQUENCY`? `TIMx` should just be `TIM4` so it's much more clear which timer we're working with?

Reply:
I set it as `TIMx_FREQUENCY` since the clock source for our timers are set up to run at 72MHz on Cube.

Reply:
That's fine with me :+1: